src/behaviors/hv-hide/index.ts

```diff
@@ -20,7 +16,9 @@ export default {
     getRoot: HvGetRoot,
     updateRoot: HvUpdateRoot,
   ) => {
-    const targetId: ?DOMString = element.getAttribute('target');
+    const targetId: DOMString | null | undefined = element.getAttribute(
```

Comment:
Did you decide to leave DOMString in the types?

Reply:
yeah I wanted to decouple the types renaming to another future PRs, to limit the size of the diff